data/en/getnumericdate.json

```diff
@@ -1,15 +1,16 @@
 {
 	"name": "getNumericDate",
 	"type": "function",
-	"syntax": "getNumericDate(date)",
+	"syntax": "GetNumericDate(arg1)",
```

Comment:
Let's keep this as lowerCamelCase (keep lowercase g)

Reply:
So we're only changing the Arguments? I do prefer lowerCamelCase, but that casing matched Lucee docs. If that is the preferred way, 'll leave the functions syntax at loweCamelCase.

Reply:
Done.